src/splitInteger.js

```diff
@@ -1,9 +1,5 @@
-/**
- * @param {number} value
- * @param {number} numberOfParts
- *
- * @returns {number[]}
- */
+'use strict';
+
 function splitInteger(value, numberOfParts) {
```

Comment:
Please, write **tests** for splitInteger function that takes 2 positive integers value and numberOfParts and returns an array containing exactly numberOfParts integer elements.

Reply:
For example: 

test(`should split a number into equal parts
  if a value is divisible by a numberOfParts`, () => {

  expect(splitInteger(6, 2))
    .toEqual([3, 3]);
});